NitroxPatcher/Patches/Persistent/CellManager_TryLoadCacheBatchCells_Patch.cs

```diff
@@ -12,21 +12,60 @@ class CellManager_TryLoadCacheBatchCells_Patch : NitroxPatch, IPersistentPatch
         public static readonly Type TARGET_CLASS = typeof(CellManager);
         public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("TryLoadCacheBatchCells", BindingFlags.Public | BindingFlags.Instance);
         
-        public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
+        public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
```

Comment:
What are all of the additional code changes to this file for?

Reply:
The additional changes in this file are so that Singleplayer can spawn items once again, Essentially it just ends up being 

```csharp
string pathPrefix = CellManager.GetCacheBatchCellsPath(Multiplayer.Active ? "" : this.streamer.tmpPathPrefix, cells.batch);
```

Reply:
for persistent patches that severely affect Singleplayer such as this one, I cobbled together this extra "helper" in order to make it easier to have persistent transpiler patches co-existing between single and multi player modes, also helps provide an example for some users trying to pick up transpilers as we all know... it is its own little section of hell